spellchecker/helpers.py

```diff
@@ -153,3 +154,21 @@ def __next__(self):
             return line[self.term_col], \
                    count, \
                    line[self.canonical_term_col] if self.canonical_term_col in line else None
+
+
+class ListIterator:
```

Comment:
Not sure if this is helping or making it more complicated than
```
spellcheck.load_dictionary([(word.strip().lower(), 1, word) for word in your_list])
```
In any case, please remove the three members that are not used.

Reply:
removing the three attributes make sense.

I prefer to keep the class tho, unless we better explain how to load the dictionary, by implementing better typehinting or explaining requirements for the word_iterator in the docstring of load_dictionary.

Putting myself into the position of someone who wish to use the library but doesn't know about it. Reading the source code should not be a requirement to understand what is needed to use it.